frontend/interfaces/host.ts

```diff
@@ -84,6 +84,38 @@ export interface IDeviceUser {
   source: string;
 }
 
+const DEVICE_USER_SOURCE_TO_DISPLAY: { [key: string]: string } = {
+  google_chrome_profiles: "Google Chrome",
+  mdm_idp_accounts: "identity provider",
+} as const;
+
+const getDeviceUserSourceForDisplay = (s: string): string => {
+  return DEVICE_USER_SOURCE_TO_DISPLAY[s] || s;
+};
+
+const getDeviceUserForDisplay = (d: IDeviceUser): IDeviceUser => {
+  return { ...d, source: getDeviceUserSourceForDisplay(d.source) };
+};
+
+/*
+ * mapDeviceUsersForDisplay is a helper function that takes an array of device users and returns a
+ * new array of device users with the source field mapped to a more user-friendly value. It also
+ * ensures that the identity provider account is always the first device user in the array.
+ */
+export const mapDeviceUsersForDisplay = (
```

Comment:
Don't we usually put this kind of thing in `helpers.tsx` instead of the interface files?

Reply:
I think they are best co-located with the interfaces since they are functions that depend on the interface as a parameter and/or return an instance of the interface. Ideally helpers would a place to put things that aren't so closely related to a particular interface.